packages/styles/src/private/makeCreateStyles.ts

```diff
@@ -0,0 +1,15 @@
+function createStyle(content: string) {
+  const style = document.createElement('style');
+  style.append(document.createTextNode(content));
+  return style;
+}
+
+export default function makeCreateStyles(...contents: string[]) {
```

Comment:
Should we call this `makeCreateStyleElements`?

Reply:
I don't see the need in extra verbosity. Let me know if you prefer more verbose one